.editorconfig

```diff
@@ -7,7 +7,7 @@ charset = utf-8
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.{yml,twig,php}]
+[*.{yaml,yml,twig,php}]
```

Comment:
Not sure how you managed to obtain such a diff… I don't understand what is happening, I mean, 3.x is already like this.

Reply:
I think you have this weird history because you forgot to use `git push --force`